File: DevExpress-Demo/CS/PivotGridDemo.Wpf/Data/FormatConditionItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PivotGridDemo {
    public enum FormatConditionType {
        DataBar
    }
    public class FormatConditionItem {
        public FormatConditionType Type { get; set; }
        public string MeasureName { get; set; }
        public string RowName { get; set; }
        public string ColumnName { get; set; }
        public string PredefinedFormatName { get; set; }
    }
}

File: DevExpress-Demo/VB/PivotGridDemo.Wpf/Data/FormatConditionItem.vb
Namespace PivotGridDemo

    Public Enum FormatConditionType
        DataBar
    End Enum

    Public Class FormatConditionItem

        Public Property Type As FormatConditionType

        Public Property MeasureName As String

        Public Property RowName As String

        Public Property ColumnName As String

        Public Property PredefinedFormatName As String
    End Class
End Namespace
